core/src/main/java/com/linecorp/armeria/client/endpoint/healthcheck/HttpHealthCheckedEndpointGroupBuilder.java

```diff
@@ -77,12 +81,26 @@ public HttpHealthCheckedEndpointGroupBuilder clientFactory(ClientFactory clientF
         return this;
     }
 
+    /**
+     * Sets the {@link Function} that customizes a {@link HttpClient} for health check.
+     * <pre>{@code
+     * new HttpHealthCheckedEndpointGroupBuilder(delegate, healthCheckPath)
+     *    .withClientOptions(op -> op.defaultResponseTimeout(Duration.ofSeconds(1)))
+     *    .build();
```

Comment:
4-space indentation? (it's 3-space currently)

Reply:
oops, fixed